### DISCUSSION TITLE
Allow pasting images

### DISCUSSION THREAD

**Discussion by meldarionqeusse** [enchant97/note-mark#214] [Ideas]
I think this app has got potential to become the most rounded note taking app but I tend to take a lot of screenshots and paste them in my notes. I am able to do this in [flatnotes](https://demo.flatnotes.io/) I was not able to do it here and I find it limiting.

**Comment by enchant97** [comment]
Excellent idea! Not sure how best to implement this at the moment. But I can definitely see the advantage.
